projects/plugins/crm/includes/jpcrm-feature-sniffer.php

```diff
@@ -97,6 +97,11 @@ public function sniff_for_plugin( $args = array(), $is_silent = false ) {
 	 *
 	 */
 	public function show_feature_alerts() {
+		// lets remove these in rebranded versions
```

Comment:
Don't we want them to see alerts about using WooSync, Mailpoet, GiveWP, etc.?

If not, it'd be better to handle this upstream so they don't run through a bunch of code they don't need by wrapping the following code in `ZeroBSCRM.Core.php`:

```
		##WLREMOVE
		$this->feature_sniffer = new JPCRM_FeatureSniffer();
		$this->jpcrm_sniff_features();
		##/WLREMOVE
```

Reply:
My thinking, and I can confirm with rebrand customers is if they're using the rebranded version they will have already used (and seen) the notices in the CRM before going the white label route.

Reply:
The admin notices were/are popping up in Jetpack CRM branding, sure we could change them over to the brand template but IMHO its cleaner to remove them under the assumption the CRM operator knows his way around the CRM (and have upgraded to our top bundle)